packages/bitcore-wallet-service/lib/blockchainexplorers/v8.js

```diff
@@ -20,187 +20,187 @@ var Constants = Common.Constants,
   Defaults = Common.Defaults,
   Utils = Common.Utils;
 
-
 function v8network(bwsNetwork) {
   if (bwsNetwork == 'livenet') return 'mainnet';
+  // Uncomment to turn on regtest
+  // if (bwsNetwork == 'testnet') return 'regtest';
```

Comment:
make this pull from config `regtestEnabled: true` where would be appropriate in config